packages/core/kind/kind.js

```diff
@@ -21,17 +21,17 @@ import styles from './styles';
 /**
  * @callback ComputedPropFunction
  * @memberof core/kind
- * @param {Object<string, any>} [props]
- * @param {Object<string, any>} [context]
+ * @param {Object<string, any>} props
+ * @param {Object<string, any>} context
```

Comment:
I think we should leave `context` optional as it will be undefined if context is not applied, I believe.  @ryanjduffy ?

Reply:
You may be right. I'll have to check again

Reply:
Looks like context is set to an empty object when `contextType` is unset or set wrong so the proposed change is correct.

Reply:
:themoreyouknow: